lib/openxml/docx/properties/container_property.rb

```diff
@@ -7,14 +7,14 @@ class ContainerProperty < BaseProperty
 
         class << self
           def child_class(*args)
-            if args.any?
-              prop_name = args.first.to_s.split(/_/).map(&:capitalize).join # LazyCamelCase
-              child_class_obj = OpenXml::Docx::Properties.const_get prop_name
-              @child_class = OpenXml::Docx::Properties.const_get prop_name
-            end
+            @child_classes = args.map do |arg|
+              prop_name = arg.to_s.split(/_/).map(&:capitalize).join # LazyCamelCase
```

Comment:
`capitalize` is part of Ruby's `String` class. I believe we're using that here instead of relying on `activesupport`.